Development-Guide/Compile-and-Flash-STMBoard-Guide/README.adoc

```diff
@@ -0,0 +1,67 @@
+= Compiling and Flashing STM Board
+
+NOTE: This has only been tested on Linux Mint, but is expected to work on other Debian based distros as well.
+
+== Setup
+
+. Go to UCSolarCarTeam repo > Recruit Resources, and clone.
+
+. Navigate to STM32F4Discovery-Resources and run the auto installer using `sudo ./install.sh`. 
+This should download the necessary tools needed (gcc4mbed, STlink, and CubeMX2Makefile)
+
+. http://www.st.com/en/development-tools/stm32cubemx.html[Go to the download page of the official ST website by clicking this link^].
+
+. Scroll to the bottom of the page to the *GET SOFTWARE* section and click *Get Software*.
+
+. Follow the instructions and download CubeMX.
+
+. Extract files from `.zip` file using a preferred method.
+For example, use `unzip file.zip -d Destination_folder`.
+
+. Navigate to the directory the files were extracted to.
+
+. Run *SetupSTM32CubeMX-VERSION.linux* using `./SetupSTM32CubeMX-{var}.linux` where  {var} is the version of CubeMx that is being installed.
+
+. Follow the instructions in the Installation Wizard.
+
+
+== Generating Code with CubeMX
+
+. Open CubeMX. The default path is `/home/STM32CubeMX`.
```

Comment:
Actually, I think the default path is `$HOME/STM32CubeMX` or `~/STM32CubeMX`.  `/home/` goes somewhere else, the proper path is also (On my computer) `/home/bill/STM32CubeMX`